src/components/Card/Card.jsx

```diff
@@ -49,6 +52,14 @@ export const CardTitle = (
   </span>
 );
 
+export const CardTitleWrapper = styled.div`
```

Comment:
We are no longer using styled components in our repository. This should be refactored to use sass. Prop values that need to be used should be passed in through CSS custom properties. Examples of this can be found [here](https://github.com/IBM/carbon-addons-iot-react/pull/885) in the `GaugeGraph.jsx` file.